scripts/linkbot/monitor-bb-sd.js

```diff
@@ -1,49 +1,29 @@
-const jsdom = require("jsdom");
-const { JSDOM } = jsdom;
-const fs = require('fs')
-const { updateURLLink } = require('./common');
+const { getCurrentTestData, getLinksByMatch, updateURLLink } = require('./common');
 
-const target = "BBB_DEBIAN_SDCARD_IMAGE_URL"
+const target = 'BBB_DEBIAN_SDCARD_IMAGE_URL';
+const reg = 'bone-debian-(?<version>[0-9]+.[0-9]+)-iot-armhf-(?<date>[0-9]{4}-[0-9]{2}-[0-9]{1,2})-4gb.img.xz$';
```

Comment:
Q: Why change to single `'`?

Reply:
that's just my local `prettier` installation, makes all my JS code _prettier_ without me worrying about a thing 😄

Reply:
G.E.N.I.U.S